bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/servlets/AdaptiveImageServlet.java

```diff
@@ -287,12 +281,14 @@ private void resizeAndStreamAsset(SlingHttpServletResponse response, ValueMap co
         }
         int rotationAngle = getRotation(componentProperties);
         Rectangle rectangle = getCropRect(componentProperties);
-        if (rotationAngle != 0 || rectangle != null || resizeWidth > 0) {
+        boolean flipHorizontally = componentProperties.get(Image.PN_FLIP_HORIZONTAL, Boolean.FALSE);
```

Comment:
@raducotescu - for the other transform properties, we rely on the `ImageResource` class, shouldn't we use the same approach for flip, rather than defining them on the core image model.

Reply:
`ImageResource` doesn't yet have properties for indicating that these transformations need to be performed. The enhancement in the TouchUI editor is recent and will probably be made available through a FP, but a FP cannot provide new API. Hence why I added these constants in our API.